common/static/common/js/discussion/views/discussion_thread_view.js

```diff
@@ -86,13 +86,6 @@
                     throw new Error("invalid mode: " + this.mode);
                 }
                 this.readOnly = $(".discussion-module").data('read-only');
-                this.model.collection.on("reset", function(collection) {
-                    var id;
-                    id = self.model.get("id");
-                    if (collection.get(id)) {
-                        self.model = collection.get(id);
-                    }
-                });
```

Comment:
@e-kolpakov I'm afraid I don't remember the state of things well enough to tell you that. It's likely-- I know since we had to re-add this a few times I probably tried to isolate as small a change as I could to get the desired effect. Don't know how much has changed since then, either.